src/sdk/namespace_fs.js

```diff
@@ -121,13 +121,15 @@ class NamespaceFS {
      *  bucket_path: string;
      *  fs_backend?: string;
      *  bucket_id: string;
+     *  namespace_resource_id: string;
```

Comment:
Since namespace_resource_id is optional you want the jsdoc to represent that -
```suggestion
     *  namespace_resource_id?: string;
```